samples/tab-add-in-combined/nodejs/README.md

```diff
@@ -0,0 +1,181 @@
+---
+page_type: sample
+description: Sample Outlook add-in and Teams tab app.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "05/15/2023 01:00:00 PM"
+urlFragment: officedev-microsoft-teams-samples-multiple-capability-discounts-ts
```

Comment:
Please add same path in Sample.json,
If it does not exist, please create it. Refer other samples in the Repository

Reply:
Will cover in different new PR

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/965